ai2thor/controller.py

```diff
@@ -461,6 +455,29 @@ def __init__(
                     event.metadata['errorMessage'])
                 )
 
+    def _build_server(self, host, port, width, height):
+
+        if self.server is not None:
+            return
+
+        if self.server_type == 'wsgi':
```

Comment:
Could we make these an `enum` or constants of a class?

Reply:
I modified the field to 'server_class' and you pass in the class which will be WsgiServer or FifoServer and we default to WsgiServer